calico/_data/navbars/getting-started.yml

```
@@ -130,6 +130,8 @@ section:
       path: /getting-started/kubernetes/vpp/getting-started
     - title: Enable IPsec encryption between nodes
       path: /getting-started/kubernetes/vpp/ipsec
+    - title: VPP implementation specificities & known-issues
```

"specificities" sounds strange in English.  Perhaps:
```suggestion
    - title: Details of VPP implementation & known-issues
```